uui/components/pickers/PickerInput.tsx

```diff
@@ -77,7 +77,11 @@ export class PickerInput<TItem, TId> extends PickerInputBase<TItem, TId, PickerI
     renderFooter() {
         const footerProps = this.getFooterProps();
 
-        return this.props.renderFooter ? this.props.renderFooter(footerProps) : <DataPickerFooter { ...footerProps } size={ this.props.size } />;
+        return this.props.renderFooter ? (
+            this.props.renderFooter(footerProps)
+        ) : (
+            <DataPickerFooter selectionMode={ this.props?.selectionMode } { ...footerProps } size={ this.props.size } />
```

Comment:
let's move 'selectionMode={ this.props?.selectionMode }' to footerProps

Reply:
Thanks, done